front/app/components/admin/PostManager/components/IdeasCount.tsx

```diff
@@ -22,89 +22,34 @@ const Container = styled.div`
   font-weight: 500;
 `;
 
-interface InputProps {
-  assignee?: string | null;
+interface Props extends Omit<UseIdeasCountProps, 'projectIds'> {
   project?: string | null;
-  phase?: string | null;
-  topics?: string[] | null;
-  ideaStatus?: string | null;
-  feedbackNeeded: boolean;
-  searchTerm?: string | null;
 }
 
-interface DataProps {
-  ideasCount: GetIdeasCountChildProps;
-}
-
-interface Props extends InputProps, DataProps {}
-
-interface State {}
+const IdeasCount = ({ project, ...otherProps }: Props) => {
+  const count = useIdeasCount({
+    ...otherProps,
+    projectIds: project ? [project] : undefined,
```

Comment:
Not a fan of explicitly setting undefined. Undefined means it's not defined, but if you set it, it's defined. If you see what I mean :-) So I rather use null in such cases. Here it may seem insignificant, but you sometimes want to check for specifically that: no one assigned it a value yet, the BE didn't return anything yet (undefined) or we gave it a 'nothing' value (null).

Anyway, the prop types of these resource hooks/components would have to be changed because they don't allow for null at the moment.